jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -701,6 +719,7 @@ class DeploymentFlow extends React.Component {
         title="Set your maximum monthly budget (optional)">
         <DeploymentExpertBudget
           budget={this.state.budget}
+          estimateWithSLA={((hourPrice * machineCount * 720) + estimate).toFixed(2)}
```

Comment:
720 magic number?

Reply:
average number of hours in a month - I'll add a comment